configuration/boot_folder.md

```diff
@@ -1,10 +1,12 @@
-# The Boot folder
+# The Boot Folder
 
-In a basic Raspbian install, the boot files are stored on the first partition of the SD card, which is VFAT-formatted. This means that it can be read on both Windows and Linux devices. 
+In a basic Raspbian install, the boot files are stored on the first partition of the SD card, which is formatted with the FAT filesystem. This means that it can be read on Windows, Mac and Linux devices.
+
+Once Linux has booted, the boot partition is mounted as /boot.
```

Comment:
I wonder if this might be better as something like "Once Raspbian has booted on the Raspberry Pi, the boot partition is mounted as `/boot`." to avoid any ambiguity? (given that the previous paragraph talks about reading the SD card on Linux)

Reply:
I've moved that sentence to after the next paragraph.

Reply:
done